OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -42,6 +42,12 @@ public sealed class WeaponInfo
 		[Desc("The sound played when the weapon is fired.")]
 		public readonly string[] Report = null;
 
+		[Desc("The sound played when the weapon is reloaded.")]
+		public readonly string[] ReloadSound = null;
```

Comment:
Please rename these to something like `AfterFireSound` and `AfterFireSoundDelay`.  These are separate to the reloading logic in the `AmmoPool`s, so the current names are misleading.